packages/design-system/src/components/combobox.tsx

```diff
@@ -82,7 +82,9 @@ export const ComboboxListboxItem = forwardRef(ListboxItemBase);
 
 export const ComboboxPopper = Popper;
 
-export const ComboboxPopperContent = PopperContent;
+export const ComboboxPopperContent = styled(PopperContent, {
+  zIndex: 10,
```

Comment:
we haven't needed it so far, why starting now? on design system level we shouldn't need zIndex

Reply:
got it, lets us $1 instead of 10
```suggestion
  zIndex: '$1',
```